flexget/plugins/plugin_configure_series.py

```diff
@@ -16,6 +16,8 @@
 log = logging.getLogger('configure_series')
 Base = db_schema.versioned_base('import_series', 0)
 
+supported_ids = FilterSeriesBase().supported_ids()
```

Comment:
this causes method to be called when module is imported and supported_ids will be set whatever happens to be loaded at that time, remove this from module level